vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution.cc

```diff
@@ -150,25 +150,31 @@ void Contribution::ResetReconcileStamp() {
 }
 
 void Contribution::StartMonthlyContribution() {
+  const auto reconcile_stamp = ledger_->GetReconcileStamp();
+  ResetReconcileStamp();
+
   if (!ledger_->GetRewardsMainEnabled()) {
-    ResetReconcileStamp();
     return;
   }
+
   BLOG(1, "Staring monthly contribution");
```

Comment:
Nit: Not really changed here, but maybe we can fix the typo: Staring -> Starting